.circleci/config.yml

```diff
@@ -3,23 +3,42 @@ version: 2.1
 workflows:
   test:
     jobs:
-      - build-run-linux
+    - build-run-linux:
+        context: hello-world-demos
```

Comment:
I've already created this CircleCI context, which provides the `LD_HELLO_WORLD_*` environment variables .